src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -224,6 +224,19 @@ public Position getPosition() {
         });
   }
 
+  @GraphQLQuery(name = "allPreviousPositions")
+  public CompletableFuture<List<PersonPositionHistory>> loadAllPreviousPositions(
+      @GraphQLRootContext Map<String, Object> context) {
+    if (previousPositions != null) {
```

Comment:
`previousPositions` is used by `loadPreviousPositions` is it safe to use the same member for also `loadAllPreviousPositions`?

Reply:
Fixed using different members.